docs/modules/operation/pages/deep-dive/user-management/user-config.adoc

```diff
@@ -0,0 +1,62 @@
+
+[[ga-user-config]]
+= User Creation and Configuration
+
+Only a user with admin privileges can create users and assign security roles to them.
+We recommend creating a new user with admin privileges instead of using the default `admin` (see <<quick-start/users.adoc#ga-admin-user-setup, admin user setup>>).
+
+Ideally, each user account corresponds to a person, to help track who performs tasks in your {page-component-title} system.
+Assigning different <<deep-dive/user-management/security-roles.adoc#ga-role-user-management-roles, security roles>> to each user helps restrict what tasks the user can perform.
+
+In addition to local users, you can configure external authentication services including link:https://opennms.discourse.group/t/spring-security-and-ldap/1425[LDAP/LDAPS], link:https://opennms.discourse.group/t/spring-security-and-radius/1424[RADIUS], and link:https://opennms.discourse.group/t/single-sign-on-sso-using-spring-security-and-kerberos/[Single Sign On (SSO]).
+Configuration specifics for these services are outside the scope of this documentation.
+
+WARNING: Do not delete the default `admin` and `rtc` users.
+         The `rtc` user is used for the communication of the Real-Time Console on the start page to calculate the node and service availability.
+
+[[ga-user-create]]
+== Create a user
+
+. Log in as a user with administrative permissions.
+. Click the gear icon in the top right.
+. Choose menu:Configure OpenNMS [Configure Users, Groups and On-Call roles > Configure Users].
+. Click *Add new user* and specify a user ID, password, password confirmation and click *OK*.
++
+WARNING: Note that you cannot use angle brackets (<>), single (') and double quotation marks ("), and the ampersand symbol (&) in the user ID.
```

Comment:
```suggestion
WARNING: You cannot use angle brackets (<>), single (') and double quotation marks ("), or the ampersand symbol (&) in a user ID.
```

Reply:
For some reason this is in italics, so I'll check locally.